src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -3838,5 +3838,227 @@ static void M()
             var compilation = CreateCompilationWithMscorlib45(source, references: new[] { reference });
             compilation.VerifyEmitDiagnostics();
         }
+
+        [Fact, WorkItem(64964, "https://github.com/dotnet/roslyn/issues/64964")]
+        public void UnobservedAwaitableExpression_AsyncTopLevelStatement()
```

Comment:
>AsyncTopLevelStatement [](http://example.com/codeflow?start=50&length=22)

Consider asserting the fact in order to avoid unintentional change to the scenario in the future.